### ISSUE TITLE
Failed to use velad install

### ISSUE THREAD

**Issue by oiar** [kubevela/kubevela#4555]
**Describe the bug**
E0804 12:28:28.875499   27279 reflector.go:138] pkg/mod/k8s.io/client-go@v0.23.6/tools/cache/reflector.go:167: Failed to watch *unstructured.Unstructured: an error on the server ("apiserver not ready") has prevented the request from succeeding (get jobs.batch)

``` shell
velad install
INFO[0000] portmapping '8090:80' targets the loadbalancer: defaulting to [servers:*:proxy agents:*:proxy]
Preparing K3s images...
Saving k3s image airgap install tarball to /Users/apple/.vela/velad/k3s/k3s-airgap-images.tgz
Successfully prepare k3s image:  /Users/apple/.vela/velad/k3s/k3s-airgap-images.tgz
Successfully prepare k3d images
Loading k3d images...
Loaded image: ghcr.io/k3d-io/k3d-proxy:5.4.1

Loaded image: ghcr.io/k3d-io/k3d-tools:5.4.1

Loaded image: rancher/k3s:v1.21.10-k3s1

Successfully load k3d images
Creating k3d cluster...
Launching k3d cluster: velad-cluster-default
INFO[0003] Prep: Network
INFO[0004] Created network 'k3d-velad'
INFO[0004] Created image volume k3d-velad-cluster-default-images
INFO[0004] Starting new tools node...
INFO[0004] Starting Node 'k3d-velad-cluster-default-tools'
INFO[0015] Creating node 'k3d-velad-cluster-default-server-0'
INFO[0015] Creating LoadBalancer 'k3d-velad-cluster-default-serverlb'
INFO[0015] Using the k3d-tools node to gather environment information
INFO[0015] Starting new tools node...
INFO[0015] Starting Node 'k3d-velad-cluster-default-tools'
INFO[0016] Starting cluster 'velad-cluster-default'
INFO[0016] Starting servers...
INFO[0016] Starting Node 'k3d-velad-cluster-default-server-0'
INFO[0035] All agents already running.
INFO[0035] Starting helpers...
INFO[0035] Starting Node 'k3d-velad-cluster-default-serverlb'
INFO[0042] Injecting records for hostAliases (incl. host.k3d.internal) and for 3 network members into CoreDNS configmap...
Successfully create k3d cluster
Successfully setup cluster
Generating host kubeconfig into /Users/apple/.kube/velad-cluster-default
Generating internal kubeconfig into /Users/apple/.kube/velad-cluster-default-internal
Successfully generate internal kubeconfig at /Users/apple/.kube/velad-cluster-default-internal
Setting kubeconfig env for VelaD...
Checking and installing vela CLI...
vela CLI is already installed, skip
Skip importing vela-core and VelaUX image on darwin-arm64
Saving and temporary helm chart file: vela-core-*.tgz
open the tar to tmpDir /Users/apple/.vela/tmp
Copy velaux-v1.5.0.tgz file to /Users/apple/.vela/addons/velaux-v1.5.0.tgz
Extracting /Users/apple/.vela/addons/velaux-v1.5.0.tgz to /Users/apple/.vela/addons/velaux
Installing vela-core Helm chart...
Executing "vela install --file /Users/apple/.vela/tmp/vela-core --detail=false --version v1.5.0 --set= --namespace=vela-system "

Check Requirements ...
Installing KubeVela Core ...
Helm Chart used for KubeVela control plane installation: /Users/apple/.vela/tmp/vela-core
I0804 12:11:43.256044   27279 apply.go:113] "creating object" name="applicationrevisions.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0804 12:11:43.541367   27279 apply.go:113] "creating object" name="applications.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0804 12:11:43.675936   27279 apply.go:113] "creating object" name="componentdefinitions.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0804 12:11:43.752385   27279 apply.go:113] "creating object" name="definitionrevisions.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0804 12:11:44.025535   27279 apply.go:113] "creating object" name="envbindings.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0804 12:11:44.084134   27279 apply.go:113] "creating object" name="healthscopes.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0804 12:11:44.178190   27279 apply.go:113] "creating object" name="manualscalertraits.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0804 12:11:44.223722   27279 apply.go:113] "creating object" name="policies.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0804 12:11:44.268310   27279 apply.go:113] "creating object" name="policydefinitions.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0804 12:11:44.342364   27279 apply.go:113] "creating object" name="resourcetrackers.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0804 12:11:44.395571   27279 apply.go:113] "creating object" name="scopedefinitions.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0804 12:11:44.490751   27279 apply.go:113] "creating object" name="traitdefinitions.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0804 12:11:44.613579   27279 apply.go:113] "creating object" name="workflows.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0804 12:11:44.804018   27279 apply.go:113] "creating object" name="workflowstepdefinitions.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0804 12:11:44.924984   27279 apply.go:113] "creating object" name="workloaddefinitions.core.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
I0804 12:11:45.024911   27279 apply.go:113] "creating object" name="rollouts.standard.oam.dev" resource="apiextensions.k8s.io/v1, Kind=CustomResourceDefinition"
Start upgrading Helm Chart kubevela in namespace vela-system
E0804 12:12:20.039480   27279 reflector.go:138] pkg/mod/k8s.io/client-go@v0.23.6/tools/cache/reflector.go:167: Failed to watch *unstructured.Unstructured: an error on the server ("apiserver not ready") has prevented the request from succeeding (get jobs.batch)
W0804 12:12:20.923289   27279 reflector.go:324] pkg/mod/k8s.io/client-go@v0.23.6/tools/cache/reflector.go:167: failed to list *unstructured.Unstructured: an error on the server ("apiserver not ready") has prevented the request from succeeding (get jobs.batch)
E0804 12:12:20.923334   27279 reflector.go:138] pkg/mod/k8s.io/client-go@v0.23.6/tools/cache/reflector.go:167: Failed to watch *unstructured.Unstructured: failed to list *unstructured.Unstructured: an error on the server ("apiserver not ready") has prevented the request from succeeding (get jobs.batch)
E0804 12:13:18.001443   27279 reflector.go:138] pkg/mod/k8s.io/client-go@v0.23.6/tools/cache/reflector.go:167: Failed to watch *unstructured.Unstructured: an error on the server ("apiserver not ready") has prevented the request from succeeding (get jobs.batch)
E0804 12:14:14.856680   27279 reflector.go:138] pkg/mod/k8s.io/client-go@v0.23.6/tools/cache/reflector.go:167: Failed to watch *unstructured.Unstructured: an error on the server ("apiserver not ready") has prevented the request from succeeding (get jobs.batch)
E0804 12:15:11.903264   27279 reflector.go:138] pkg/mod/k8s.io/client-go@v0.23.6/tools/cache/reflector.go:167: Failed to watch *unstructured.Unstructured: an error on the server ("apiserver not ready") has prevented the request from succeeding (get jobs.batch)
E0804 12:20:52.016270   27279 reflector.go:138] pkg/mod/k8s.io/client-go@v0.23.6/tools/cache/reflector.go:167: Failed to watch *unstructured.Unstructured: an error on the server ("apiserver not ready") has prevented the request from succeeding (get jobs.batch)
E0804 12:24:37.570335   27279 reflector.go:138] pkg/mod/k8s.io/client-go@v0.23.6/tools/cache/reflector.go:167: Failed to watch *unstructured.Unstructured: an error on the server ("apiserver not ready") has prevented the request from succeeding (get jobs.batch)
W0804 12:25:35.170820   27279 reflector.go:324] pkg/mod/k8s.io/client-go@v0.23.6/tools/cache/reflector.go:167: failed to list *unstructured.Unstructured: an error on the server ("apiserver not ready") has prevented the request from succeeding (get jobs.batch)
E0804 12:25:35.170863   27279 reflector.go:138] pkg/mod/k8s.io/client-go@v0.23.6/tools/cache/reflector.go:167: Failed to watch *unstructured.Unstructured: failed to list *unstructured.Unstructured: an error on the server ("apiserver not ready") has prevented the request from succeeding (get jobs.batch)
E0804 12:26:32.435821   27279 reflector.go:138] pkg/mod/k8s.io/client-go@v0.23.6/tools/cache/reflector.go:167: Failed to watch *unstructured.Unstructured: an error on the server ("apiserver not ready") has prevented the request from succeeding (get jobs.batch)
E0804 12:27:29.186366   27279 reflector.go:138] pkg/mod/k8s.io/client-go@v0.23.6/tools/cache/reflector.go:167: Failed to watch *unstructured.Unstructured: an error on the server ("apiserver not ready") has prevented the request from succeeding (get jobs.batch)
E0804 12:28:28.875499   27279 reflector.go:138] pkg/mod/k8s.io/client-go@v0.23.6/tools/cache/reflector.go:167: Failed to watch *unstructured.Unstructured: an error on the server ("apiserver not ready") has prevented the request from succeeding (get jobs.batch)
E0804 12:29:28.632860   27279 reflector.go:138] pkg/mod/k8s.io/client-go@v0.23.6/tools/cache/reflector.go:167: Failed to watch *unstructured.Unstructured: an error on the server ("apiserver not ready") has prevented the request from succeeding (get jobs.batch)
```
``` shell
velad status
Checking cluster status...
K3d images status:
 ✔ image rancher/k3s:v1.21.10-k3s1 ready
 ✔ image ghcr.io/k3d-io/k3d-tools:5.4.1 ready
 ✔ image ghcr.io/k3d-io/k3d-proxy:5.4.1 ready
Cluster(K3d) status:
 ✘ cluster [default] not ready: Failed to get helm releases: Kubernetes cluster unreachable: an error on the server ("apiserver not ready") has prevented the request from succeeding
```
**To Reproduce**

**Expected behavior**

**Screenshots**

**KubeVela Version**
``` shell
vela version
CLI Version: v1.4.2
Core Version:
GitRevision: git-eb9ddaa
GolangVersion: go1.17.11
```

**Docker Version**
``` shell
docker version
Client:
 Cloud integration: v1.0.24
 Version:           20.10.14
 API version:       1.41
 Go version:        go1.16.15
 Git commit:        a224086
 Built:             Thu Mar 24 01:49:20 2022
 OS/Arch:           darwin/arm64
 Context:           default
 Experimental:      true

Server: Docker Desktop 4.8.2 (79419)
 Engine:
  Version:          20.10.14
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.16.15
  Git commit:       87a90dc
  Built:            Thu Mar 24 01:45:44 2022
  OS/Arch:          linux/arm64
  Experimental:     false
 containerd:
  Version:          1.5.11
  GitCommit:        3df54a852345ae127d1fa3092b95168e4a88e2f8
 runc:
  Version:          1.0.3
  GitCommit:        v1.0.3-0-gf46b6ba
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

**Cluster information**

**Additional context**
The error is shown in the figure above.

**Comment by chivalryq** [comment]
What is the velad version?

**Comment by chivalryq** [comment]
Investigating, you can walk around as follows:
1. Use [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) to setup cluster.
2. download latest Vela CLI: `curl -fsSl https://static.kubevela.net/script/install-vela.sh | bash`
3. run `vela install`

**Comment by oiar** [comment]
> Can it be fixed by velad uninstall and velad install?

No.

And my docker is running on amd64 MAC, I find some error log in container <k3d-velad-cluster-default-serverlb>.

```sh
2022/08/04 11:52:39 [emerg] 90#90: io_setup() failed (38: Function not implemented)
```

**Comment by chivalryq** [comment]
> Your docker is running on amd64 MAC
But your docker client and server's ARCH are arm64. Do you misspell the word?

**Comment by oiar** [comment]
> > Your docker is running on amd64 MAC
> 
> But your docker client and server's ARCH are arm64. Do you misspell the word?

You're right.

**Comment by chivalryq** [comment]
Will fixed here: jhttps://github.com/kubevela/velad/pull/76

**Comment by oiar** [comment]
> Will fixed here: [kubevela/velad#76](https://github.com/kubevela/velad/pull/76)

I see. Thank you.

**Comment by chivalryq** [comment]
This has been fixed, you can run installation script to install newest version. :) @oiar
